study.md

```diff
@@ -29,21 +29,23 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop is the sequence of how JavaScript handles normal javascript functions, callback functions, and events.
```

Comment:
Good, the event loop watches the call stack, if it is empty it will push something from the queue to the stack.